package.json

```diff
@@ -69,26 +69,26 @@
     "coverage": "http-server -c-1 -o -p 9875 ./coverage"
   },
   "dependencies": {
-    "@angular/animations": "5.2.1",
-    "@angular/common": "5.2.1",
-    "@angular/core": "5.2.1",
-    "@angular/forms": "5.2.1",
-    "@angular/http": "5.2.1",
-    "@angular/platform-browser": "5.2.1",
-    "@angular/platform-browser-dynamic": "5.2.1",
-    "@angular/platform-server": "5.2.1",
-    "@angular/router": "5.2.1",
+    "@angular/animations": "^5.2.5",
```

Comment:
I notice all these dependencies are being changed to a caret (`^`), which trusts minor and point releases to not be breaking. Are we sure we want that?  I thought we ran into issues with that in the past.

Reply:
Since we switched to yarn, that should no longer be a problem. Yarn manages our dependencies, and ensures everybody gets the exact same version. So if a new version of a dependency is released it won't automatically update, even if package.json allows it. Yarn also manages the versions in package.json, if you `yarn add` a dependency it will be added with a caret by default. This allows you to use the `yarn upgrade` command later, to upgrade to the most recent version the caret syntax allows.

Reply:
Thanks for the explanation, @artlowel. I wasn't aware of that.

This seems like something we could add a few notes about in our README, perhaps in this section: https://github.com/DSpace/dspace-angular#3rd-party-library-installation  That way it's clearer to others how we recommend updating / managing these dependencies.

Based on that, this all looks good. I should be able to give it a test today and report back on results.

Reply:
Oops... just realized this *is* in the README. I need to read closer :)  Here's where there's notes about `yarn upgrade`: https://github.com/DSpace/dspace-angular#yarn-lockfile